manim/mobject/text/tex_mobject.py

```diff
@@ -381,6 +381,29 @@ def set_color_by_tex(self, tex, color, **kwargs):
             part.set_color(color)
         return self
 
+    def set_opacity_by_tex(self, tex, opacity=0.5, remaining_opacity=None, **kwargs):
+        """
+        Sets the opacity of the tex specified. If 'remaining_opacity' is specified,
+        then the remaining tex will be set to that opacity.
+
+        Parameters
+        ----------
+        tex : str
+            The tex to set the opacity of.
+        opacity : float, optional
+            Default 0.5. The opacity to set the tex to
+        remaining_opacity : float, optional
```

Comment:
The type hints should be moved from the docstring to the method head: `tex` -> `tex: str`, `opacity` -> `opacity: float = 0.5`, etc.


```suggestion
        tex
            The tex to set the opacity of.
        opacity
            Default 0.5. The opacity to set the tex to
        remaining_opacity
```

Reply:
Sorry it took me a while to get to this. I agree with both changes mentioned. Unfortunately, the auto-formatter that runs during commit moved the parameters onto their own line. Not a fan of this personally but it is required to pass the checks. Let me know what you think of the changes. Thanks!